docs/userGuide/syntax/questions.md

```diff
@@ -242,7 +242,7 @@ and not others.
 {% set blanksQuestion %}
 <question type="blanks" hint="Google it!">
 
-  ##### German sociologist __________ called the process of simultaneously analyzing the behavior of individuals and the society that shapes that behavior __________.
+  <md>German sociologist __________ called the process of simultaneously analyzing the behavior of individuals and the society that shapes that behavior __________.</md>
```

Comment:
Hi, thank you for improving the style of the questions;
I noticed that these two examples sit side by side in the user doc with the purpose of demonstrating the effect of "no-intermediate-result attribute". So it feel abrupt to have style. Could you help make them consistent?
 
<img width="803" alt="Screenshot 2024-03-04 at 12 55 46" src="https://github.com/MarkBind/markbind/assets/121547057/d083a596-5867-4b54-9bb2-2dbe7b8ed70d">
<img width="806" alt="Screenshot 2024-03-04 at 12 55 53" src="https://github.com/MarkBind/markbind/assets/121547057/2664f614-9d3f-4a80-aa27-3b193b980b06">

Reply:
Sorry for not noticing! I've since standardised the style of the questions to make it consistent.